doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_architecture_pm.rst

```diff
@@ -250,3 +244,105 @@ Power management benchmark
 **************************
 
 To benchmark the power consumption in *Idle* state, see :ref:`multicore_idle_test`.
+
+Power management optimization example
+*************************************
+
+This example focuses on a HID device use case (for example, a mouse)
+In this use case the typical workflow is the following:
+
+  1. Sample input data from sensors.
+  2. Construct a HID report.
+  3. Transmit the report over the radio.
+  4. Sleep until the next period (usually, less than 1 ms).
+
+The power consumption targets for this example are the following:
+
+* Sleep currents (5 V supply):
+
+  * Application core (Suspend-to-RAM): < 10 µA
+  * Radio core (Suspend-to-Idle, cache disabled): < 10 µA
+
+* Active currents (estimates at 5 V):
+
+  * Radio core in WFI: ~0.25 mA
+  * App core in WFI: ~0.15 mA
+  * Core wake-up every 1 ms: +0.05 mA
+  * ADC sample every 1 ms: +0.10 mA
+  * SPI transaction every 1 ms: +0.20 mA
+  * IPC message every 1 ms: +0.15 mA
+
+Recommended Kconfig configuration
+=================================
+
+Set these Kconfigs as follows for the application running on the application core:
+
+  * CONFIG_PM=y
+  * CONFIG_PM_S2RAM=y
+  * CONFIG_POWEROFF=y
+  * CONFIG_PM_S2RAM_CUSTOM_MARKING=y
+  * CONFIG_PM_DEVICE=y
+  * CONFIG_PM_DEVICE_RUNTIME=y
+
+Set these Kconfigs as follows for the application running on the radio core:
+
+  * CONFIG_PM=y
+  * CONFIG_POWEROFF=y
+  * CONFIG_PM_DEVICE=y
+  * CONFIG_PM_DEVICE_RUNTIME=y
+
+Consider also the following:
+
+  * Disable all unused peripherals before entering sleep (Zephyr's API does this automatically when supported).
+  * Add `zephyr,pm-device-runtime-auto` in the DTS for all peripherals with runtime PM support.
+    ## DTS to JSON update here
+  * Build and program an empty image on any unused core to release shared resources.
+    ##sample of this empty image?
+
+Deep-sleep policy
+-----------------
+
+Minimum sleep durations:
+
+  * Suspend-to-Idle: ≥ 1 ms
+  * Suspend-to-RAM: ≥ 2 ms
+
+Use policy locks to prevent premature deep-sleep entry:
+
+.. code-block:: c
+
+   /* In active mode: lock deep states */
+   pm_policy_state_lock_get(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
+   pm_policy_state_lock_get(PM_STATE_SUSPEND_TO_RAM, PM_ALL_SUBSTATES);
+
+   /* Before going to sleep: unlock */
+   pm_policy_state_lock_put(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
+   pm_policy_state_lock_put(PM_STATE_SUSPEND_TO_RAM, PM_ALL_SUBSTATES);
+
+Memory and cache optimizations
+------------------------------
+
+  * Relocate code and data to TCM to avoid waking MRAM.
+  * Profile L1/L2 cache usage to minimize MRAM accesses.
+  * Disable MRAM latency manager (`CONFIG_MRAM_LATENCY=n`) to power MRAM off when idle.
+
+Peripheral and clock recommendations
+------------------------------------
+
+  * Use slow-domain (13X) peripherals to avoid waking the fast domain.
```

Comment:
```suggestion
  * Use global peripherals from the slow domain (peripheral index 13X) to avoid waking up the fast global domain, which uses more power.
```